frontends/web/src/locales/en/app.json

```diff
@@ -411,6 +411,11 @@
         "p1": "When you buy bitcoin via Pocket, you are using an external service. This service is out of scope of the BitBox02 Security Threat model and relies on the safety and security of the environment which the BitBoxApp software is running in. However we work together to improve security by using a two factor authentication mechanism to verify the address you are receiving to.",
         "title": "Security model"
       },
+      "usedAddress": "The address {{address}} has been already used, please start again.",
+      "verifyBitBox02": {
+        "dialog": "Please verify that the address you received via email matches the one displayed on your device. If possible, you should open the email on a second device for better security.",
+        "title": "Verify address on BitBox02"
```

Comment:
There is already `receive.verifyBitBox02` ith the same text, maybe use that instead?